/pathfinder:
  spin_rate: 15
  cones_max_distance: 1.5
  odom_topic: "/pose"
  cones_topic: "/cones"
  use_laserscan: true
  laser_topic: "/laser"
  cmdvel_topic: "/rossae/cmdvel"
  path_topic: "/pathfinder/path"
  clustering_threshold: 0.5
  publish_path: true
  num_of_path_points: 5
  time_interval: 0.2


